src/components/Level/index.js

```diff
@@ -11,21 +11,30 @@ import {
   Dino,
   TeamBoard,
 } from './styled';
+import { dinos } from '../data/dinos.js';
+import { legends } from '../data/legends.js';
+import { levels } from '../data/levels.js';
 import Legend from '../Legend';
 import Counter from '../Counter';
 import counterReducer from '../Counter/reducer';
 import trianglify from 'trianglify';
-import { ReactComponent as OnePointRedDino } from '../../icons/dinosaurs/red-one-point.svg';
-import { ReactComponent as OnePointBlueDino } from '../../icons/dinosaurs/blue-one-point.svg';
 import './Level.css';
+import shuffle from 'shuffle-array';
 
 const randBetween = (low, high) => {
   return Math.floor(low + Math.random() * (high - low));
 };
 
 const Level = () => {
-  const { difficulty } = useParams();
-
+  const { stage } = useParams();
+  const difficulty = window.location.href
```

Comment:
Once you make the changes to the routes above, destructure difficulty from useParams instead.
`const { state, difficulty } = useParams()`